### PR TITLE
Version compatibility chart

### PR DESCRIPTION
**Pull request by yorinasub17**
See https://gruntwork-io.slack.com/archives/CJMPH63E0/p1560174442009200

### CODE REVIEW

File: content/reference/version-compatibility/index.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ the package version listed is the last version to support the specific Terraform
...
 <!-- This was generated using the Markdown Table Generator: https://www.tablesgenerator.com/markdown_tables -->
 
-|  IaC Package                | Terraform 0.11.X | Terraform 0.12.X |
-|-----------------------------|------------------|------------------|
-| Terragrunt                  | v0.18.7          | >=v0.19.0        |
-| Terratest                   | ALL              | >=v0.16.0        |
-| package-terraform-utilities | v0.0.8           | >=v0.1.0         |
+| IaC Package                                                                                | Terraform 0.11.X | Terraform 0.12.X |
+|--------------------------------------------------------------------------------------------|------------------|------------------|
+| [Terragrunt](https://github.com/gruntwork-io/terragrunt)                                   | v0.18.7          | >=v0.19.0        |
```
Comment (brikis98) [inline content/reference/version-compatibility/index.md]: Should it be `<= v0.18.7`?

File: content/reference/version-compatibility/index.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ the package version listed is the last version to support the specific Terraform
...
+| [Terratest](https://github.com/gruntwork-io/terratest)                                     | ALL              | >=v0.16.0        |
+| [package-terraform-utilities](https://github.com/gruntwork-io/package-terraform-utilities) | v0.0.8           | >=v0.1.0         |
```
Comment (brikis98) [inline content/reference/version-compatibility/index.md]: `<= v0.0.8`?

Comment (yorinasub17) [reply content/reference/version-compatibility/index.md]: Updated

File: content/reference/version-compatibility/index.md (hunk 1 above)
Comment (yorinasub17) [reply content/reference/version-compatibility/index.md]: Updated

**Comment by yorinasub17** [comment]
Ok going to merge and deploy this. Will paste link here once done.

**Comment by yorinasub17** [comment]
https://docs.gruntwork.io/reference/version-compatibility/

**Comment by robmorgan** [comment]
Nice 👍

**Pull request merged**